src/sqltest/parts/notebook/common.ts

```diff
@@ -55,46 +55,48 @@ export class NotebookModelStub implements INotebookModel {
 	get contentChanged(): Event<NotebookContentChange> {
 		throw new Error('method not implemented.');
 	}
-    get specs(): nb.IAllKernels {
-        throw new Error('method not implemented.');
-    }
-    get contexts(): IDefaultConnection {
-        throw new Error('method not implemented.');
-    }
-    get providerId(): string {
-        throw new Error('method not implemented.');
-    }
-    get applicableConnectionProviderIds(): string[] {
-        throw new Error('method not implemented.');
-    }
-    changeKernel(displayName: string): void {
-        throw new Error('Method not implemented.');
-    }
-    changeContext(host: string, connection?: IConnectionProfile, hideErrorMessage?: boolean): Promise<void> {
-        throw new Error('Method not implemented.');
-    }
-    findCellIndex(cellModel: ICellModel): number {
-        throw new Error('Method not implemented.');
-    }
-    addCell(cellType: CellType, index?: number): void {
-        throw new Error('Method not implemented.');
-    }
-    deleteCell(cellModel: ICellModel): void {
-        throw new Error('Method not implemented.');
-    }
-    saveModel(): Promise<boolean> {
-        throw new Error('Method not implemented.');
-    }
-    pushEditOperations(edits: ISingleNotebookEditOperation[]): void {
-        throw new Error('Method not implemented.');
-    }
-    getApplicableConnectionProviderIds(kernelName: string): string[] {
-        throw new Error('Method not implemented.');
-	}
-	get onValidConnectionSelected(): Event<boolean>
-	{
+	get specs(): nb.IAllKernels {
 		throw new Error('method not implemented.');
 	}
+	get contexts(): IDefaultConnection {
```

Comment:
Did a tab indentation for the whole file since it's got both space and tabbed ones.